x/interchaintxs/keeper/msg_server.go

```diff
@@ -50,7 +50,8 @@ func (k Keeper) RegisterInterchainAccount(goCtx context.Context, msg *ictxtypes.
 
 	icaOwner := ictxtypes.NewICAOwnerFromAddress(senderAddr, msg.InterchainAccountId)
 
-	if err := k.icaControllerKeeper.RegisterInterchainAccount(ctx, msg.ConnectionId, icaOwner.String()); err != nil {
+	// FIXME: empty version string doesn't look good
+	if err := k.icaControllerKeeper.RegisterInterchainAccount(ctx, msg.ConnectionId, icaOwner.String(), ""); err != nil {
```

Comment:
And this?

Reply:
That's ok, cause inside ibc-go a metadata is being set to default value in case version is an empty string: https://github.com/cosmos/ibc-go/blob/ecb845d5e43f53decf48f8ed88c7847a9a4375cb/modules/apps/27-interchain-accounts/controller/keeper/handshake.go#L45

But I'll keep the comment, cause maybe we will want some custom metadata in the future